Desktop version

leonardodiasb opened the pull request:
Created Desktop version in a media query from the mobile first version css.
Added a list with links in the navbar.
Changed content by duplicating the html on the work section, adding a display "none" for the "works-section-desktop" on mobile CSS part, and display "none" for the "works-section" in the media query.
Applied hover function to each container on the works section.
Applied CSS on the other sections.

leonardodiasb committed:
Add background images to the image folder. Finish Headline CSS section

leonardodiasb committed:
Finish work section CSS desktop style

leonardodiasb committed:
Finish first half of About section. Include about myself title, subtitle, let's connect, social icons and get my resume button position

leonardodiasb committed:
Finish Contact Me section and Navbar CSS. Add hover functionality to works container. Change Works section content.

leonardodiasb committed:
Correct linter errors

vichuge commented on style.css:
```diff
@@ -406,3 +426,283 @@ textarea[type="text"] {
+  display: none;
+}
+
+@media (min-width: 500px) {
```
- [ ] Please, use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px), for example `@media screen and (max-width: 1080px) {` 🐶

vichuge commented on index.html:
```diff
@@ -17,12 +17,17 @@
+            </ul>
         </nav>
     </header>
     <section id="headline">
```
- [ ] The background image on this section appears at the middle of the screen on desktop size, please check this little but important issue.
![image](https://user-images.githubusercontent.com/20563252/121597615-8979af80-ca06-11eb-9e26-b44a57a4d9f5.png)

vichuge reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @leonardodiasb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 
- No Linters errors ✔️ 
- Good job for design ✔️ 

## Aspects to improve ♻️

- [ ] It looks like the form has the mobile size on desktop, please fix this
![image](https://user-images.githubusercontent.com/20563252/121598219-348a6900-ca07-11eb-9fdd-1c7c80816359.png)

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- Use more details according to Figma style, image below:
![image](https://user-images.githubusercontent.com/20563252/121598513-7fa47c00-ca07-11eb-8342-0c1ccbc8b98a.png)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

leonardodiasb commented:
Hello Victor ! On this first image of the Contact Me section I am pretty sure I corrected it on this Desktop-Version branch as you can see here:
![reply1](https://user-images.githubusercontent.com/61919617/121600908-9523a100-ca1b-11eb-8f67-7748197f698c.png)
![reply2](https://user-images.githubusercontent.com/61919617/121600933-9f459f80-ca1b-11eb-8d81-92293512f847.png)

And I also added those background images:
![reply3](https://user-images.githubusercontent.com/61919617/121601149-ec297600-ca1b-11eb-8f30-3dc851ec0067.png)
![raply4](https://user-images.githubusercontent.com/61919617/121601156-efbcfd00-ca1b-11eb-9cbc-223db26e3367.png)

I'm sorry for asking, but did I send the url from the last commit I did today on Microverse Dashboard ?

leonardodiasb committed:
Change  min-width on media query to 1080px

Shaher-11 reviewed (approved):
Hello @leonardodiasb ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point Is optional to implement however it's highly recommended!_:blush:

- Add some space between the about me paragraph and let's conncet

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏
